afm/server.py

```diff
@@ -56,13 +57,16 @@ def _filter_columns(self, schema, columns):
         return pa.schema([pa.field(f.name, f.type, f.nullable, f.metadata) for f in fields])
 
     # write arrow dataset to filesystem
-    def _write_asset(self, asset, reader):
+    def _write_asset(self, asset, reader, write_mode):
         # in this implementation we currently begin by reading the entire dataset
         record_batches = reader.read_all().combine_chunks().to_batches()
         transformed_batches = transform_batches(asset.actions, record_batches)
-        # currently, write_dataset supports the parquet format, but not csv
-        ds.write_dataset(transformed_batches, base_dir=asset.path, format=asset.format,
-                         filesystem=asset.filesystem)
+        if write_mode == "append":
```

Comment:
Can you please add some explanation? You can use parts of the following:
```
 If the client request to append to the existing data then the flag `overwrite_or_ignore` is used with a basename_template that is related to the time of writing. This operation writes the data to a new file while ignoring (not changing) the existed files.
 Otherwise, the flag `delete_mathing` is used. This operation writes the data to a new file while deleting the existed files.
```

Reply:
Sure. Done.